docs/README.md

```diff
@@ -48,15 +57,15 @@ Now we will run KubeArmor as a systemd process on the Open Horizon Agent VM
    sudo apt install ./kubearmor_0.5.5_linux-amd64.deb
    ```
 
-> Note that the above automatically installs `bpfcc-tools` with our package, but your distribution might have an older version of BCC. In case of errors, consider installing `bcc` from [source](https://github.com/iovisor/bcc/blob/master/INSTALL.md#source).
+> Note that the above automatically installs `bpfcc-tools` with our package, but your distribution might have an older version of BCC. In case of errors, consider installing `bcc` from [source ](https://github.com/iovisor/bcc/blob/master/INSTALL.md#source){:target="_blank"}{: .externalLink}.
 
 ---
 <details><summary> Note: For distributions other than Ubuntu/Debian</summary>
 <p>
 
-i. Refer [Installing BCC](https://github.com/iovisor/bcc/blob/master/INSTALL.md#installing-bcc) to install pre-requisites.
+i. Refer [Installing BCC ](https://github.com/iovisor/bcc/blob/master/INSTALL.md#installing-bcc){:target="_blank"}{: .externalLink} to install pre-requisites.
 
-ii. Download release tarball from KubeArmor [releases](https://github.com/kubearmor/KubeArmor/releases)
+ii. Download release tarball from KubeArmor [releases ](https://github.com/kubearmor/KubeArmor/releases){:target="_blank"}{: .externalLink}
```

Comment:
> [releases ]

[releases]

Reply:
The extra space is intentional. It will separate the linked text from the launch glyph.